Header header
SkeletonJoint joint
geometry_msgs/Vector3 basePoint
geometry_msgs/Vector3 closestPoint
float32[] distances
